fido2/hid/linux.py

```diff
@@ -0,0 +1,73 @@
+# Original work Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Modified work Copyright 2020 Yubico AB. All Rights Reserved.
+# This file, with modifications, is licensed under the above Apache License.
+
+
+from __future__ import absolute_import
+
+from .base import HidDescriptor, FileCtapHidConnection, parse_report_descriptor
+
+import glob
+import fcntl
+import struct
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+GET_INFO = 0x80084803
+GET_DESC_SIZE = 0x80044801
+GET_DESC = 0x90044802
+
```

Comment:
Are those HIDIOCGRAWINFO, HIDIOCGRDESCSIZE and HIDIOCGRDESC? Curious about the reason not to use the names from hidraw.h. Can those ever change?

Reply:
Good point. I'll rename them to use the hidraw.h names.